tox.ini

```diff
@@ -14,6 +14,10 @@ commands = py.test --timeout 300 []
 install_command = python -m pip install {opts} {packages}
 usedevelop = True
 
+[testenv:coverage-py3]
+basepython = python3
+commands = py.test --timeout 300 --cov=pip --cov-report=term-missing --cov-report=xml --cov-report=html tests/unit {posargs}
```

Comment:
``tests/unit {posargs}`` -> ``{posargs:tests/unit}`` maybe?

Reply:
Hi @cryvate! :)

I don't think in the near future anyone would want to pass any directory other for measuring coverage. pip's integration tests spawn subprocesses, who won't get tracked in the coverage metrics.